Source/WebCore/PAL/pal/text/TextCodecSingleByte.cpp

```diff
@@ -180,22 +180,21 @@ template<const SingleByteDecodeTable& decodeTable> SingleByteEncodeTable tableFo
 {
     // Allocate this at runtime because building it at compile time would make the binary much larger and this is often not used.
     static constexpr auto size = std::size(decodeTable) - std::count(std::begin(decodeTable), std::end(decodeTable), replacementCharacter);
-    static const SingleByteEncodeTableEntry* entries;
+    static LazyNeverDestroyed<std::array<SingleByteEncodeTableEntry, size>> entries;
```

Comment:
Does this make the binary larger the way the comment says we should not?

Reply:
I think you're right, I'll use a raw pointer.